File: fixtures/hub-cobol-clbs-mini/batch/reference_emit_portreadrn.java
// EXPECTED: 4
// Reference emit for PORTREADRN — PORTREAD LINE SEQ record count.
public final class PortreadrnRef {
  public static void main(String[] args) {
    String[][] rows = {
      {"PORT0001", "A"}, {"PORT0002", "A"}, {"PORT0003", "S"}, {"PORT0004", "A"}
    };
    System.out.println(rows.length);
  }
}

File: fixtures/hub-cobol-clbs-mini/batch/reference_emit_portreadrn.cs
// EXPECTED: 4
// Reference emit for PORTREADRN — PORTREAD LINE SEQ record count.
using System;
public static class PortreadrnRef {
  public static void Main() {
    var rows = new (string Id, string Status)[] {
      ("PORT0001", "A"), ("PORT0002", "A"), ("PORT0003", "S"), ("PORT0004", "A")
    };
    Console.WriteLine(rows.Length);
  }
}
